### ISSUE TITLE
IJPOpen

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PaDS#20]
James, it may be you just haven't set up the subscriptions for IJPOpen yet, but I wanted to call attention to this because we are planning to go live Monday, if there aren't any show stopper issues.

But @rickardnyman please confirm with DT, but my understanding from before is that:

1) IJPOpen has its own subscription, which is free
2) All PEPArchive users get IJPOpen automatically.

Subscription config 2 does not seem to be working.  But @rickardnyman please confirm or check with DT that No. 2 is one of the settings.

Test case: User test4 tries to read an IJPOpen document and gets:

<img width="860" alt="image" src="https://user-images.githubusercontent.com/4837554/150240300-3dec8a05-e925-44e0-b2bf-9e2777b592ae.png">

**Comment by nrshapiro** [comment]
Maybe something else is up.  My tests are failing.  I set up one to use my neilshapironet user which is IJPOpen only and it passed this weekend, but now fails.  Interactively retesting, I see:

<img width="1332" alt="image" src="https://user-images.githubusercontent.com/4837554/150242208-e42e5b3a-65ab-455c-82c4-2a5117054d6c.png">

**Comment by jwoosnam** [comment]
I am not sure if point 2 above is correct. At present PaDS PEP Archive subscribers only have access to IJP Open as abstracts.  I will wait for David to respond.
If required it is a simple change to the PaDS content sets.

There was a recent change to allow PaDS content sets to make use of the newish AccessClassification field.  The IJP Open content set was set to Archive only.  I have now changed it to Special.

**Comment by nrshapiro** [comment]
@jwoosnam 

David responded: 
  "All pepweb subscribers see IJPO."

(By that he means PEPArchive, and "see" means full access.  He doesn't really want them to be able comment, but that's another big discussion for later.)

And...My IJPOpen only account is working again, thanks.

**Comment by jwoosnam** [comment]
I have adjusted the product/content sets on PaDS **Stage** so that PEP users have full read access to any issue of the IJP Open "Journal" documents, if they have a subscription to any of the following products:
Pepweb3y
PEPWeb
PEPComp1
PEPComp
PEPWEBS
Pwi
Pwr

I have a script to replicate this on PaDS production which will need to be run after the new PEP version is pushed to production.
@nrshapiro @davidtuckett @AthenaPEP

**Comment by davidtuckett** [comment]
To be clear. How this was meant to work - reolicatig the old system -was as follows:
1. All active logged pepweb subscribers can view the papers.
2. I dont mind if  commentary is also visible.
2. But only IJP Open subscribers can comment.
I have agreed with Neil that if we cant do this now we look at it later.

**Comment by jwoosnam** [comment]
I am sorry David this is still not completely clear. The list of PaDS products I put in my previous comment are all the products which give access to Archive (non-embargoed) documents on PEP. When you say "PEPweb subscribers" I think you mean all of these people, not just the people who are subscribed to the product "PEPweb", which would just be the group subscriptions, for instance various Psychoanalytic Societies.
Therefore, for now, I think everything is working as you want in a "Read Only" version.

I expect that in due course the commentary bit will require PaDS to send something in the PEP user information to say that the user is a "IJOD IJP Open Discussion" subscriber.  No-one has mentioned that to me yet.

**Comment by davidtuckett** [comment]
Yes. Your list defines pepweb subscribers ie those who should have access to Archive (non-embargoed) documents on PEP. Noted re commentary,
D
